packages/concerto-cto/lib/parser.pegjs

```diff
@@ -1397,6 +1398,43 @@ LongFieldDeclaration
       return result;
     }
 
+MapDeclaration
+    = decorators:Decorators __ MapToken __ id:Identifier __
+    "{" __ body:MapDeclarationBody __ "}"
+    {
+      const result = {
+        $class: "concerto.metamodel@1.0.0.MapDeclaration",
+        name:   id.name,
+        properties:  body.declarations,
+        ...buildRange(location())
+      };
+      if (decorators.length > 0) {
+        result.decorators = decorators;
+      }
+      return result;
+    }
+
+MapDeclarationBody
+  = decls:MapPropertyDeclaration* {
```

Comment:
This looks like a "zero or more" whereas I'd expect the Map declaration to have two child elements: key and value.